### PR TITLE
Improved how tables are parsed by the importer

### LINKED ISSUE #169: Importer not detecting existing tables

### PR DESCRIPTION
**Pull request by MikeStower**
Resolves #169

### CODE REVIEW

File: Presentation/WindowsClient/Models/TaskRunners/ApsimXporter.cs [hunk 1]
```diff
@@ -181,7 +181,6 @@ private async Task<IModel> CreateExperiment(string name, int id)
                             b_interception = 1,
                             c_interception = 0,
                             d_interception = 0,
-                            soil_albedo = 0.13,
```
Comment (jbrider) [inline Presentation/WindowsClient/Models/TaskRunners/ApsimXporter.cs:184]: what does this have to do with importing?

Comment (MikeStower) [reply Presentation/WindowsClient/Models/TaskRunners/ApsimXporter.cs:184]: It doesn't, fixing this was a side-effect of resolving the earlier merge conflict.
